src/scenes/Languages/Overview/LanguageOverview.graphql

```diff
@@ -0,0 +1,41 @@
+query Language($input: ID!) {
+  language(id: $input) {
+    ...languageDetails
+  }
+}
+
+fragment languageDetails on Language {
+  id
+  createdAt
+  name {
+    value
+    canEdit
+  }
+  displayName {
+    value
+    canEdit
+  }
+  beginFiscalYear {
+    value
+    canEdit
+  }
+  ethnologueName {
+    value
+    canEdit
+  }
+  ethnologuePopulation {
+    value
+    canEdit
+  }
+  organizationPopulation {
+    value
+    canEdit
+  }
+  rodNumber {
+    value
+    canEdit
+  }
+  #    TODO add these in back later when API accepts these
+  #    sensitivity
+  #    avatarLetters
```

Comment:
Is avatar letters still not working?

Reply:
I'm getting an error in the playground when I try to query avatar letters about it not existing on type project.

Reply:
It's working for me. Is your API code up to date?